lib/parsers/asm-parser.ts

```diff
@@ -709,6 +715,18 @@ export class AsmParser extends AsmRegex implements IAsmParser {
                     labels: labelsInLine,
                 });
             }
+
+            match = line.match(this.relocationRe);
+            if (match) {
+                const address = parseInt(match.groups.address, 16);
+                const relocname = match.groups.relocname;
+                const relocdata = match.groups.relocdata;
+                const match_value = relocdata.match(this.relocDataSymNameRe); // TO REMOVE
```

Comment:
this has `//TO REMOVE` - should it be removed before merging? If not, can we file an issue and track it?

Reply:
Will document

Reply:
Grrr, this one was already removed but the push failed yesterday and I didn't notice.